x/escrow/msg_test.go

```diff
@@ -280,59 +258,54 @@ func TestUpdateEscrowMsg(t *testing.T) {
 	b := weavetest.NewCondition()
 	c := weave.NewCondition("monkey", "gelato", []byte("berry"))
 
-	cases := []struct {
+	cases := map[string]struct {
 		msg   *UpdatePartiesMsg
-		check checkErr
+		check error
 	}{
-		// nothing
-		0: {
+		"nothing": {
 			&UpdatePartiesMsg{
 				Metadata: &weave.Metadata{Schema: 1},
 			},
-			errors.ErrInput.Is,
+			errors.ErrInput,
 		},
-		// proper: valid id, one valid permission
-		1: {
+		"proper: valid id, one valid permission": {
 			&UpdatePartiesMsg{
 				Metadata: &weave.Metadata{Schema: 1},
 				EscrowId: escrow,
 				Source:   a.Address(),
 			},
-			noErr,
+			nil,
 		},
-		// valid escrow, no permissions
-		2: {
+		"valid escrow, no permissions": {
 			&UpdatePartiesMsg{
 				Metadata: &weave.Metadata{Schema: 1},
 				EscrowId: escrow,
 			},
-			errors.ErrEmpty.Is,
+			errors.ErrEmpty,
 		},
-		// invalid escrow, proper permissions
-		3: {
+		"invalid escrow, proper permissions": {
 			&UpdatePartiesMsg{
 				Metadata: &weave.Metadata{Schema: 1},
 				EscrowId: scarecrow,
 				Source:   a.Address(),
 			},
-			errors.ErrInput.Is,
+			errors.ErrInput,
 		},
-		// allow multiple permissions
-		4: {
+		"allow multiple permissions": {
 			&UpdatePartiesMsg{
 				Metadata:    &weave.Metadata{Schema: 1},
 				EscrowId:    escrow,
 				Destination: b.Address(),
 				Arbiter:     c.Address(),
 			},
-			noErr,
+			nil,
 		},
 	}
 
 	for i, tc := range cases {
-		t.Run(fmt.Sprintf("case-%d", i), func(t *testing.T) {
+		t.Run(fmt.Sprintf("case-%s", i), func(t *testing.T) {
```

Comment:
Please rename `i` to `testName` and use it as the test name. No need for the `case-` prefix.